app/renderer/components/main/braveryPanel.js

```diff
@@ -262,55 +271,63 @@ class BraveryPanel extends ImmutableComponent {
           compactBraveryPanel && styles.braveryPanel_compact__stats
         )}>
           <div data-test-id='adsBlockedStat'
-            onClick={this.onToggleAdsAndTracking}
```

Comment:
now that onClick handler is set for `<span>` the data-test-id should be moved over there as well. Otherwise, some tests that depend on it will perma fail because it is clicking an element that has no listener for changes.